src/app/components/tfrs/tfrs.component.html

```diff
@@ -1 +1,92 @@
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project Start After</mat-label>
+        <input matInput [matDatepicker]="picker1" [formControl]="startAfterDate" >
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matIconSuffix [for]="picker1"></mat-datepicker-toggle>
+        <mat-datepicker #picker1></mat-datepicker>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project End Before</mat-label>
+        <input matInput [matDatepicker]="picker2" [formControl]="endBeforeDate">
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matIconSuffix [for]="picker2"></mat-datepicker-toggle>
+        <mat-datepicker #picker2></mat-datepicker>
+      </mat-form-field>
+
+        <div></div>
+
+      <section>
+        <div class="example-button-row">
+          <button mat-raised-button color="primary" (click)="getProjects()">Search</button>
+        </div>
+      </section>
+    </div>
+</div>
+
+
+<div class="example-container mat-table">
+
+<table mat-table [dataSource]="projectList" matSort (matSortChange)="announceSortChange($event)"  class="mat-elevation-z8 mat-table">
+    
+  <ng-container matColumnDef="name">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+    <td mat-cell *matCellDef="let element"> {{element.name}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="start_date" >
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Start Date </th>
+    <td mat-cell *matCellDef="let element"> {{element.start_date.split('T')[0]}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="end_date">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> End Date </th>
+    <td mat-cell *matCellDef="let element"> {{element.end_date.split('T')[0]}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="status">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+    <td mat-cell *matCellDef="let element" [ngClass]="{'delivered':element.status=='DELIVERED',
+                                                      'inprogress':element.status=='INPROGRESS',
+                                                      'archived':element.status=='ARCHIVED',
+                                                      'draft':element.status=='DRAFT'}"> {{element.status}} </td>
+  </ng-container>
+
+  
+  <ng-container matColumnDef="link">
+    <th mat-header-cell *matHeaderCellDef> action </th>
+    <td mat-cell *matCellDef="let element">  
+      <a routerLink="/tfr/{{element.id}}" routerLinkActive="active" ariaCurrentWhenActive="page">View Details </a>
+    </td>
+  </ng-container>
+
+  <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+  <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+</table>
+
+
+
+  <mat-paginator [pageSizeOptions]="[3, 5, 10]"
```

Comment:
Do not give page size in HTML because in future if we wanted to change page size need to go HTML and search for page size option instead create a variable in ts file and assign if in future we wanted to change page size we can do it easily